libs/utils/executor.py

```diff
@@ -44,7 +44,7 @@
 
 class Executor():
 
-    def __init__(self, test_env, tests_conf=None):
+    def __init__(self, test_env, experiments_conf=None):
```

Comment:
Should not be mandatory?
Don't remember anymore why we used optional values at the beginning but they seems not being more an option... form the checks on lines 80:91 we do not allow a None value, isn't it?